### PR TITLE
interfaces: explain how udev-based security works

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by niemeyer** [comment]
Thanks for the docs! LGTM!

Can we get the $SNAPPY_APP reference fixed?

**Comment by zyga** [comment]
I'll coordinate with the security team to make the required changes to ubuntu-core-launcher.

File: interfaces/security.go [hunk 1]
```diff
@@ -104,7 +104,18 @@ func (sc *secComp) footerForApp(snapName, appName string) []byte {
...
 // NOTE: This interacts with ubuntu-core-launcher.
-// TODO: Explain how this works (security).
+//
+// This tag is picked up by /lib/udev/rules.d/80-snappy-assign.rules which in
+// turn runs /lib/udev/snappy-app-dev script, which re-configures the device
+// cgroup at /sys/fs/cgroup/devices/snappy.$SNAPPY_APP for the acl "c
+// $major:$minor rwm" for character devices and "b $major:$minor rwm" for block
+// devices.
+//
+// The control group is created by ubuntu-app-launcher.
```
Comment (zyga) [inline interfaces/security.go]: s/ubuntu-app-launcher/ubuntu-core-launcher/

Comment (sergiusens) [inline interfaces/security.go]: `s/ubuntu-core-launcher/snap-launcher/` :wink:

Comment (zyga) [inline interfaces/security.go]: Oh, nice, thanks, I'll correct this in a moment.

**Comment by mvo5** [comment]
Looks good except for s/ubuntu-app-launcher/ubuntu-core-launcher/ (as was already pointed out). The name "snap-launcher" is not used yet.

**Comment by zyga** [comment]
I'm closing this because of the discussed launcher work that will invalidate it.

**Pull request closed without merging**